packages/api/helper/mailer.js

```diff
@@ -64,7 +64,23 @@ function verifyHcEmail(nomination) {
     .catch(console.error);
 }
 
+function sendHIPAAEmail(nomination) {
+  email.send({
+    template: 'hipaa',
+    message: {
+      from: 'Keep Swimming Foundation <info@keepswimmingfoundation.org>',
+      replyTo: 'info@keepswimmingfoundation.org',
+      to: nomination.representativeEmailAddress,   // -> double check this is the correct email we should be using.
```

Comment:
did you check with Product on this? If so then this is good to go 🚀

Reply:
I double checked with Product. Everything should be good now. I made a few minor updates since last time.